lms/djangoapps/discussion_api/api.py

```diff
@@ -255,9 +255,12 @@ def get_thread_list(
         view=None,
         order_by="last_activity_at",
         order_direction="desc",
+        discussion_id=None,
 ):
     """
     Return the list of all discussion threads pertaining to the given course
+    (or if discussion_id is provided then return threads list against that
```

Comment:
Can remove this from the docstring here since the comment here is generic.  Not all the possible filters are listed here.  They are listed below.  You can look at `topic_id_list` as an example.